releases/release-1.8/README.md

```diff
@@ -0,0 +1,43 @@
+# Kubeflow 1.8
+
+### Links
+
+- [Release Team](release-team.md)
+- [Calendar](https://arrik.to/kf-release-team-cal)
+- [Meeting Notes](https://arrik.to/kf-release-team-notes)
+- [Recordings](https://arrik.to/kf-release-team-recordings)
+- [Retrospective Document](https://bit.ly/kf-release-1-6-retro)
+- Project Board
+- Contact
+  - [#release](https://app.slack.com/client/T7QLHSH6U/C9V2WT2KV) on slack
+  - [@release-team](https://github.com/orgs/kubeflow/teams/release-team) on GitHub
+
+### TL;DR
+
+The 1.8 release cycle is proposed as follows
+
+- **Friday, May 5th 2023**     : Week 0      - Release team is formed
+- **Monday, May 8th 2023**     : Week 1  C\* - Release cycle begins
+- **Friday, May 12th 2023**    : Week 1  C\* - Roadmaps are finalized
+- **Wednesday, Aug 16th 2023** : Week 15 C\* - Feature freeze
+- **Wednesday, Aug 21st 2023** : Week 16     - Manifests testing and bug fixing starts
```

Comment:
To better understand the proposal, what's the reason behind the 1-week gap between feature freeze and testing? Should we start testing on Aug 17 to get more time for testing and bug fixing?

Reply:
I believe the gap is there to ensure the PRs to sync manifest to Manifest repo are all merged within that timeframe.

Reply:
Exactly, feature freeze means everything in a dev state should be finalised by then, after the feature freeze, the release team alongside the WGs should ensure everything is merged in the kubeflow/manifests repo.

Reply:
Maybe adding that as a separate item would make it more verbose?

Reply:
Yeah, let me clarify that.